### PR TITLE
Dr 24 01 dmg 11c

### PR DESCRIPTION
**Pull request by thea-m**

### CODE REVIEW

File: HalleDMG/DMGYi135IIIC11c.xml [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DR" when="2024-01-18">Created record after consultation of the manuscript on 21.12.2023</change>
+        </revisionDesc>
+    </teiHeader>
+    <text xml:base="https://betamasaheft.eu/">
+        <body>
+            <div type="edition">
+                <div type="textpart" xml:lang="ar" corresp="#ms_i1.1">
```
Comment (eu-genia) [inline HalleDMG/DMGYi135IIIC11c.xml:176]: "am" 1.2?

**Review by eu-genia — CHANGES_REQUESTED** [review]

**Review by CarstenHoffmannMarburg — APPROVED** [review]

File: HalleDMG/DMGYi135IIIC11c.xml (hunk 1 above)
Comment (thea-m) [reply HalleDMG/DMGYi135IIIC11c.xml:176]: Sorry, I don't understand what you mean - 1.1 is Arabic, 1.2 Amharic

Comment (eu-genia) [reply HalleDMG/DMGYi135IIIC11c.xml:176]: Yes but which are you editing below?

Comment (thea-m) [reply HalleDMG/DMGYi135IIIC11c.xml:176]: both

**Pull request merged**